include/eld/Script/Plugin.h

```diff
@@ -230,6 +234,7 @@ class Plugin {
   std::string PluginLibraryName;
   std::string PluginType;
   std::string PluginOptions;
+  std::unordered_map<std::string, std::string> PluginStats;
```

Comment:
Please do not use unordered_map here. I would like the plugin stats to remain in the same order as the plugin inserts them.

Reply:
There are two major benefits of a consistent order of the plugin stats:

1) The stats in the right order can be more readable/easy-to-follow. For example, Input file count stat naturally precedes input sections count stat.
2) Less noise when comparing the map file diffs. If inconsistent order is used, then we may see false-positive diffs when diffing map-files.